test/inductor/test_cpp_wrapper.py

```diff
@@ -243,7 +243,13 @@ class BaseTest(NamedTuple):
         BaseTest("test_bmm1"),
         BaseTest("test_bmm2"),
         BaseTest("test_cat"),  # alias
-        BaseTest("test_convolution1"),
+
+        # TODO: Re-enable this test after fixing cuda wrapper for conv Triton templates with dynamic shapes.
```

Comment:
@desertfire FYI I disabled this test in cpp wrapper as dynamic shape doesn't work well for Conv Triton templates.

Reply:
Ok to me.